packages/dd-trace/src/plugins/cache.js

```diff
@@ -4,6 +4,13 @@ const StoragePlugin = require('./storage')
 
 class CachePlugin extends StoragePlugin {
   static get operation () { return 'command' }
+
+  startSpan (options) {
```

Comment:
I think this part might be a little confusing. Looking across the caching/mysql/tedious PRs: In some classes, startSpan takes two arguments, but in cache related classes, it only takes one?

Or, is the change to make it a single argument going to happen everywhere? There are so many PRs I'm losing track of some of this stuff 😅

Reply:
Maybe as a stop-gap we can rename this method to `startSpanImpliedOperation(options)` and have it still call `super.startSpan`? With a TODO to refactor `startSpan` everywhere to have the same arity?

Reply:
That sounds better for continuity, will do.

Reply:
That name doesn't really help with understanding IMO. Also, the point of the `startSpan` method is explicitly to allow overriding it in every plugin.